### PR TITLE
[4.0] Help Admin Menu item should always display

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes
Forcing Help admin menu to display for all users with access to administrator as already done for Control Panel ( https://github.com/joomla/joomla-cms/pull/25012 ) 
This is already the case for 3.x. No reason to change.

### Testing Instructions
Log in backend as a manager.

### Before patch
Screen Shot 2019-05-28 at 12 22 38

### After patch
Screen Shot 2019-05-28 at 12 21 57

### CODE REVIEW

**Comment by brianteeman** [comment]
Makes sense to me

File: administrator/modules/mod_menu/Menu/CssMenu.php [hunk 1]
```diff
@@ -353,6 +353,19 @@ protected function preprocess($parent)
...
+			elseif ($item->link === 'index.php?option=com_cpanel&view=help')
+			{
+				if ($this->params->get('showhelp', 1))
+				{
+					continue;
+				}
+				// Exclude help menu item if set such in mod_menu
+				else
+				{
+					$parent->removeChild($item);
```
Comment (Quy) [inline administrator/modules/mod_menu/Menu/CssMenu.php]: Just a preference, lines 365-366 don't need to be in `else` so one less level/indentation.

Comment (infograf768) [reply administrator/modules/mod_menu/Menu/CssMenu.php]: That would not work for a superuser where it is absolutely necessary. I suggest you test.

Comment (Quy) [reply administrator/modules/mod_menu/Menu/CssMenu.php]: The logic is still the same without having to be in the `else`. Here is the suggested code:
```
			elseif ($item->link === 'index.php?option=com_cpanel&view=help')
			{
				if ($this->params->get('showhelp', 1))
				{
					continue;
				}

				// Exclude help menu item if set such in mod_menu
				$parent->removeChild($item);
				continue;
			}
```

Comment (Quy) [reply administrator/modules/mod_menu/Menu/CssMenu.php]: Superusers are affected by the `Help Menu` setting too.

Comment (infograf768) [reply administrator/modules/mod_menu/Menu/CssMenu.php]: Was lost in translation. Modified as suggested.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 2cf2f820b67cc253b2e00876d7f1d968b5a65210<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25028">issues.joomla.org/tracker/joomla-cms/25028</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on cec83df2c7a2a878d0609aa2bc821a0c01650df1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25028">issues.joomla.org/tracker/joomla-cms/25028</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by infograf768** [comment]
@wilsonge
Please merge as I have to merge branch again and again.

**Comment by wilsonge** [comment]
Honestly I don't see why the help item has to display - but if it's the case in 3.x I'm not going to spend time fighting that war 🤷‍♂

**Comment by richard67** [comment]
@wilsonge It makes users feeel safer to have a help item ;-)

**Comment by wilsonge** [comment]
We give them one by default - i don't see why we tell them our one is the compulsory one. surely that's the entire point of the customizable admin menus?

**Comment by infograf768** [comment]
@wilsonge
It is customizable through admin menu parameters. It's not only all specific joomla links.

**Pull request merged**